contracts/src/rollup/RollupCore.sol

```diff
@@ -93,7 +95,7 @@ abstract contract RollupCore is IRollupCore, PausableUpgradeable {
     address public stakeToken;
     uint256 public minimumAssertionPeriod;
 
-    mapping(address => bool) public isValidator;
+    EnumerableSetUpgradeable.AddressSet validators;
```

Comment:
note that this shifted 1 slot, but is ok because bold rollup is a new deployment